go-backend/db/recognition_hi5.go

```diff
@@ -0,0 +1,74 @@
+package db
+
+import (
+	"context"
+	logger "github.com/sirupsen/logrus"
+	"time"
```

Comment:
you guys need to run `go fmt` once. Code is not properly formatted

Reply:
we executed go fmt, but there was not any change in this particular import